maps/lv522_chances_claim.json

```diff
@@ -22,7 +22,7 @@
         580,
```

Comment:
Area Issues
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/5ef41a51-7d36-4c1d-a1e4-3db79fc51db3)
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/abdec39b-52c0-4aee-ba06-a62f4316cf0b)
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/bdc0e726-0973-462a-b4c8-a2fbad72186f)
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/351f3745-fef5-468e-bcb1-eb89783653e3)

Hull walls, doesnt seem intended.
![bilde](https://github.com/cmss13-devs/cmss13/assets/59782240/9a92ba8d-4c36-4d71-bbb8-c7c50edbbf1e)

Also a lot of areas seem to lack light fixtures 
Imagine this comment is on the .dmm file :D